src/flexmeasures_client/client.py

```diff
@@ -97,6 +97,12 @@ async def request(
         - the client polling timed out (as indicated by the client's self.polling_timeout)
         """  # noqa: E501
         url = self.build_url(uri, path=path)
+        url_msg = f"url: {url}"
+        json_msg = f"url: {url}"
+        method_msg = f"method: {method}"
+        logging.debug(url_msg)
+        logging.debug(json_msg)
+        logging.debug(method_msg)
 
         headers = await self.get_headers(include_auth=include_auth)
```

Comment:
It might be interesting to log this header, as well.

Reply:
added the header logging